### PR TITLE
Hitting ESC closes Preferences window now

### PR DESCRIPTION
**Pull request by MarcinOrlowski**

### CODE REVIEW

**Review by BFH-ktt1 — CHANGES_REQUESTED** [review]
Could you remove the System.out.print("Prefs\n"); ?

I'm not sure about the implementation, as there is a special keyhandler object that also assigns functions to for example the escape key. It might interfere, it might not.

**Comment by MarcinOrlowski** [comment]
`print()` removed.

**Review by maehne — APPROVED** [review]
Thanks @MarcinOrlowski for the contribution! Looks good to me! @BFH-ktt1 do you agree?

File: src/main/java/com/cburch/logisim/gui/prefs/PreferencesFrame.java [hunk 1]
```diff
@@ -38,9 +38,11 @@
...
 import java.awt.BorderLayout;
 import java.awt.Container;
-import javax.swing.JFrame;
-import javax.swing.JScrollPane;
-import javax.swing.JTabbedPane;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.awt.event.KeyAdapter;
+import java.awt.event.KeyEvent;
+import javax.swing.*;
```
Comment (maehne) [inline src/main/java/com/cburch/logisim/gui/prefs/PreferencesFrame.java:45]: Personally, I prefer that we import only the classes, we actually need.

Comment (maehne) [reply src/main/java/com/cburch/logisim/gui/prefs/PreferencesFrame.java]: Thanks!

**Comment by MarcinOrlowski** [comment]
Not sure how to make @BFH-ktt1 requested change resolved.

**Comment by MarcinOrlowski** [comment]
Not sure how to mark @BFH-ktt1 requested change resolved.

**Review by maehne — APPROVED** [review]

**Comment by maehne** [comment]
Let's merge this. If we should remark later a regression, we can revert it easily.

**Comment by BFH-ktt1** [comment]
@maehne : Could you please revert this pull request?
I did not have the time yet to look into the details, but each window has a key-handler. In my earlier implementation for the auto-labler I was not aware yet from this handler and implemented it directly like here as extra handler, causing some keystrokes to mal function (see one of the issues). This is why I did not merge these yet.

**Comment by MarcinOrlowski** [comment]
@BFH-ktt1 mind pointing to the code that you said you had conflict with previously?

**Pull request merged**